### PR TITLE
ofi: patch v4.1.x with some common code from master

### PR DESCRIPTION
**Pull request by hppritcha**
partial patch of 9f1081a07ac3c7b7277a27277ed970ed713207c9 to v4.1.x
to resolve a compile issue

Signed-off-by: Howard Pritchard <hppritcha@gmail.com>

### CODE REVIEW

File: ompi/mca/mtl/ofi/mtl_ofi_component.c [hunk 1]
```diff
@@ -267,6 +267,8 @@ ompi_mtl_ofi_component_register(void)
                                     MCA_BASE_VAR_SCOPE_READONLY,
                                     &ompi_mtl_ofi.num_ofi_contexts);
 
+    opal_common_ofi_register_mca_variables(&mca_mtl_ofi_component.super.mtl_version);
```
Comment (hkuno) [inline ompi/mca/mtl/ofi/mtl_ofi_component.c:270]: This is actually a comment about the header -- should there be a "Copyright (c)" line added at the top? (I couldn't seem to add a comment where there wasn't a code change.)

Comment (jsquyres) [reply ompi/mca/mtl/ofi/mtl_ofi_component.c:270]: @hppritcha You have LANL copyrights in the header; I don't know if you need to add Triad copyrights, too.

Comment (hppritcha) [reply ompi/mca/mtl/ofi/mtl_ofi_component.c:270]: missed that with the cherry-pick of a cherry-pick, updating

**Comment by hppritcha** [comment]
per discussion at today's DEv meeting we want all of the original include/exclude cherry-pick - 9f1081a so closing this PR.

**Pull request closed without merging**